service/worker/migration/activities.go

```diff
@@ -298,8 +298,8 @@ func (a *activities) UpdateActiveCluster(ctx context.Context, req updateActiveCl
 	return err
 }
 
-func (a *activities) ListWorkflows(ctx context.Context, request *workflowservice.ListWorkflowExecutionsRequest) (*listWorkflowsResponse, error) {
-	resp, err := a.frontendClient.ListWorkflowExecutions(ctx, request)
+func (a *activities) ListWorkflows(ctx context.Context, request *workflowservice.ScanWorkflowExecutionsRequest) (*listWorkflowsResponse, error) {
```

Comment:
This is not a backwards-compatible change. Any activity tasks that are on the wire when workers are upgraded will now have undefined behavior when the workers try to deserialize the activity task input. Please ensure this change is backwards-compatible